src/flows/pipes/RemoteCSV/view.tsx

```diff
@@ -0,0 +1,89 @@
+import React, {FC, useContext} from 'react'
+
+import {
+  Dropdown,
+  ComponentColor,
+  Input,
+  InputType,
+  ComponentStatus,
+} from '@influxdata/clockface'
+import {PipeProp} from 'src/types/flows'
+import {PipeContext} from 'src/flows/context/pipe'
+
+const SampleCSVs = {
+  Sample_Air_Sensor_Data:
+    'https://raw.githubusercontent.com/influxdata/influxdb2-sample-data/master/air-sensor-data/air-sensor-data-annotated.csv',
+  NOAA_Weather_Data: 'https://influx-testdata.s3.amazonaws.com/noaa.csv',
```

Comment:
So this data isn't updated regularly. IMO, we might exclude it for now since 99% of the time, a user will select it with -1h option and get 0 results.